src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlSecurityTokenHandler.ReadToken.cs

```diff
@@ -5,6 +5,8 @@
 using System.Text;
 using System.Xml;
 using Microsoft.IdentityModel.Logging;
+using Microsoft.IdentityModel.Tokens.Experimental;
+using Microsoft.IdentityModel.Tokens.Saml.Experimental;
```

Comment:
I think we should have just one experimental namespace M.IM.T.Experimental or  M.IM.Experimental.

Reply:
Will follow up with this issue in the feature branch.